projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -354,7 +354,7 @@ public static function parse( $attributes, $content ) {
 			$url                     = apply_filters( 'grunion_contact_form_form_action', "{$url}#contact-form-{$id}", $GLOBALS['post'], $id );
 			$has_submit_button_block = str_contains( $content, 'wp-block-jetpack-button' );
 			$form_classes            = 'contact-form commentsblock';
-			$form_accessible_name    = $attributes['formTitle'] ? $attributes['formTitle'] : $post->post_title;
+			$form_accessible_name    = isset( $attributes['formTitle'] ) && ! empty( $attributes['formTitle'] ) ? $attributes['formTitle'] : $post->post_title;
```

Comment:
You should be able to simplify this, if you check for `empty` you do not need to check for `isset`:

```suggestion
			$form_accessible_name    = ! empty( $attributes['formTitle'] ) ? $attributes['formTitle'] : $post->post_title;
```